2. Web API now calls Microsoft Graph/TodoListService/appsettings.json

```diff
@@ -3,6 +3,7 @@
     "Instance": "https://login.microsoftonline.com/",
     "ClientId": "[Enter_client_ID_Of_TodoListService-v2_from_Azure_Portal,_e.g._2ec40e65-ba09-4853-bcde-bcb60029e596]",
     "ClientSecret": "[Enter_client_secret_as_added_fom_the_certificates_&_secrets_page_from_your_app_registration]",
+    "CallbackPath" :  "",
```

Comment:
This is important so that the redirect URI registered in the README.md is indeed the root of the web API (and therefore the static page)